build.sbt

```diff
@@ -590,18 +590,40 @@ lazy val serialversionRemoverPlugin =
 lazy val serialversionRemoverPluginSettings = Seq(
   Compile / scalacOptions ++= (
       if (scalaVersion.value.startsWith("3."))
-        Seq("-Xplugin:" + (serialversionRemoverPlugin / Compile / Keys.`package`).value.getAbsolutePath.toString)
+        Seq("-Xplugin:" + (serialversionRemoverPlugin / Compile / Keys.`package`).value.getAbsolutePath)
       else Nil
     ))
 
-def akkaModule(name: String): Project =
-  Project(id = name, base = file(name))
+lazy val username = System.getenv("USER")
+lazy val fortifyLicense = new File(s"/Users/$username/.lightbend/fortify.license")
+
+lazy val doesFortifyLicenseExist: Boolean = {
+  import java.nio.file.Files
+  val exists = Files.exists(fortifyLicense.toPath)
+  println(s"doesCredentialExist: ($fortifyLicense) " + exists)
+  exists
+}
+
+def fortifySettings(name: String) =
+  Seq(
+    addCompilerPlugin(("com.lightbend" %% "scala-fortify" % "1.0.21").cross(CrossVersion.patch)),
```

Comment:
(Might be nice to keep the version in `/project/Dependencies.scala`, but not essential)